agent_configs/Store-Assistant.json

```diff
@@ -181,18 +181,20 @@
                         "api_schema": {
                             "url": "https://api.open-meteo.com/v1/forecast?current=temperature_2m,wind_speed_10m&hourly=temperature_2m,relative_humidity_2m,wind_speed_10m",
                             "method": "GET",
-                            "query_params_schema": {
-                                "latitude": {
-                                    "type": "number",
-                                    "description": "The latitude coordinate of the location (e.g., 52.52 for Berlin)",
-                                    "dynamic_variable": "",
-                                    "constant_value": ""
-                                },
-                                "longitude": {
-                                    "type": "number",
-                                    "description": "The longitude coordinate of the location (e.g., 13.41 for Berlin)",
-                                    "dynamic_variable": "",
-                                    "constant_value": ""
+                            "path_params_schema": {
+                                "properties": {
```

Comment:
do not nest it within properties. It needs to top level! see https://elevenlabs.io/docs/api-reference/tools/create#response.body.tool_config.WebhookToolConfig.api_schema.path_params_schema